include/openthread.h

```diff
@@ -1933,7 +1933,7 @@ OTAPI uint8_t OTCALL otGetRouterIdSequence(otInstance *aInstance);
 OTAPI ThreadError OTCALL otGetRouterInfo(otInstance *aInstance, uint16_t aRouterId, otRouterInfo *aRouterInfo);
 
 /**
- * The function retains diagnostic information for a Thread Router as parent.
+ * The function returns diagnostic information for a Thread Router as parent.
```

Comment:
nit: How about `returns` -> `retrieves` since this function returns `ThreadError`?